app/graphql/mutations/destroy_project.rb

```diff
@@ -0,0 +1,20 @@
+module Mutations
+  class DestroyProject < BaseMutation
+    include GraphqlErrors
+
+    argument :input, Types::Inputs::DestroyProjectInput, required: true
+
+    type Types::Payloads::DestroyProjectPayload
+
+    def resolve(input:)
+      project = Project.find(input.id)
+      result = Projects::Destroy.call(project: project, project_params: input.to_h, user: current_user)
```

Comment:
Do you have project params here?

Reply:
I think that isn't necessary

Reply:
Then why you you put project params to interactor context?

Reply:
it was redundant